releases/release-5.0.0.md

```diff
@@ -1,466 +1,466 @@
----
-title: What's New in TiDB 5.0
----
-
-# What's New in TiDB 5.0

```

Comment:
This file had CRLF line endings, which broke the linter. I fixed it to LF, but that's creating the large diff.